bazel/grpc_build_system.bzl

```diff
@@ -30,6 +30,13 @@ load("@build_bazel_rules_apple//apple:ios.bzl", "ios_unit_test")
 # The set of pollers to test against if a test exercises polling
 POLLERS = ["epollex", "epoll1", "poll"]
 
+def if_windows(a):
+    return select({
+        "//:windows": [],
+        "//:windows_msvc": [],
+        "//conditions:default": a,
```

Comment:
Typo? This looks like an implementation for `if_not_windows`.

Reply:
Ugh, yes, thanks.

Reply:
Fixed.